draft-ietf-tls-esni.md

```diff
@@ -375,53 +375,65 @@ This document also defines the "ech_required" alert, which clients MUST send
 when it offered an "encrypted_client_hello" extension that was not accepted by
 the server. (See {{alerts}}.)
 
-## Incorporating Outer Extensions {#outer-extensions}
+## Encoding the Inner ClientHello {#encoding-inner-clienthello}
```

Comment:
```suggestion
## Encoding ClientHelloInner {#encoding-inner-clienthello}
```

Reply:
Applied manually since I needed to rebase.